dartsim/src/JointFeatures.cc

```diff
@@ -324,6 +326,94 @@ void JointFeatures::SetJointMaxEffort(
   joint->setForceUpperLimit(_dof, _value);
 }
 
+/////////////////////////////////////////////////
+
```

Comment:
remove empty line between the separator line and function definition. Same for the other functions below
```suggestion
```